src/components/MainContent/MainContent.jsx

```diff
@@ -0,0 +1,21 @@
+import "./main-content.css";
+import { NoteInput, PinnedNotes, OtherNotes } from "../index";
+export const MainContent = () => {
+  return (
+    <div className="main-content-container">
+      {/* <label for="searchbar">
+        <input
+          class="input-corner-rounded input-md"
+          type="text"
+          name="searchbar"
+          id="searchbar"
+          placeholder="Search..."
+        />
+      </label> */}
```

Comment:
Can remove this commented section if not needed since you have already splitted into components

Reply:
> Can remove this commented section if not needed since you have already splitted into components

Done